src/NUnitFramework/framework/Attributes/SetCultureAttribute.cs

```diff
@@ -29,8 +29,15 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// Summary description for SetCultureAttribute.
+    /// Sets the current Culture for the duration of a test.
     /// </summary>
+    /// <remarks>
+    /// It may be specified at the level of a test or a fixture.
+    /// The culture remains set until the test or fixture completes and is then reset to its original value.
+    /// <para />
+    /// If you wish to use the current culture setting to decide whether to run a test, use the <see cref="CultureAttribute"/> instead of this one.
+    /// </remarks>
```

Comment:
Two uncertainties here...

1. Where do remarks show up? If they don't come up when I right-click, I don't see any advantage to having them - better to expand the summary. We've discussed this before but I can't remember the resolution - I _think_ we decided not to use Remarks. @nunit/framework-team ?

2. I don't know about putting usage info like "If you wish to use..." in the Intellisense. Seems like it gets in the way, giving the reader something extra to look at each time and figure out if they are in the right place **Set**Culture seems pretty clear to me.

Reply:
1. `<remarks>` are invisible to IntelliSense unless you are missing the `<summary>`, in which case it replaces it. Since we require `<summary>`, the Object Browser is the only place in the IDE that you can see `<remarks>`, and hypothetically if we generated website content it would show up there.
    
    If the information is interesting to the general user, I'd say it should be in the summary because putting it in `<remarks>` gives the false impression that people are going to see it and benefit from it. If we agree on this I can add this rationale to our best practices page. I want to know what others think.

 2. I'm actually amenable to this kind of documentation. Not that the user doesn't know what <code>**Set**Culture</code> means, but just a way to notify you that the other thing exists if that interests you. I could go either way.

Reply:
1. If we were to generate website content, then I'd be in favor of putting the first `para` in the summary and the second in remarks.  In fact, that's what I suggest we do here. 

As an experienced user, I'm frequently annoyed at Microsoft for telling me "obvious" things, without the ability to turn off the stream of advice. I don't want us to do that.

If the Intellisense has a sentence suggesting I may not be using the correct attribute, I'm going to pause for a few seconds, read it and think about it. I don't want to do that in this case. I think that comparisons with other attributes belong in the docs. Warnings about use of an attribute only belong here in the case where that use is __usually wrong__.  And, of course, if an attribute is usually used incorrectly, we shold probably change it!

Reply:
> I think that comparisons with other attributes belong in the docs.

Which in your temporary hypothesis earlier you assumed was generated from XML docs...

Reply:
> I think that comparisons with other attributes belong in the docs.

If we went the direction of generating website content from XML docs, which would it be? Comparison in the docs = website = XML docs, or not?

Reply:
Yes. I think that would be a good direction.

But please see my comment about what I think should be done with the current PR in the first paragraph of my last comment. Let's resolve this PR rather than getting into general discussions about the website here... that's a separate matter and doesn't actually involve me.

Reply:
Ah, I mixed up which sections you meant. I don't think that's how `<para>` is meant to be used, but I could be wrong.  I've only ever seen it used to contain each paragraph as contents.

Anyway, I'm good with that. @RalfKoban Does that make sense?

Reply:
@jnm2  Yes, I'm OK with that.